### PR TITLE
Rename truffle-kredits to kredits-contracts

### PR DESCRIPTION
**Pull request by fsmanuel**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,7 +51,7 @@
     "eslint-plugin-ember": "^5.0.0",
     "ethers": "3.0.15",
     "kosmos-schemas": "^1.1.2",
-    "kredits-contracts": "github:67P/kredits-contracts",
+    "kredits-contracts": "^3.x",
```
Comment (skddc) [inline package.json]: I think the `^` and `.x` are doing the same thing. So we should use either one of those. Personally, I like `3.x` a bit better as it's more expressive.

Comment (skddc) [reply package.json]: Weird: I already changed this and it's `3.x` in master, so I'm not sure what's going on here.

Comment (fsmanuel) [reply package.json]: #metoo

**Comment by fsmanuel** [comment]
closing in favor of #69

**Pull request closed without merging**